godot-core/src/builtin/callable.rs

```diff
@@ -515,6 +547,8 @@ mod custom_callable {
 
         /// `None` if the callable is multi-threaded ([`Callable::from_sync_fn`]).
         pub(super) thread_id: Option<ThreadId>,
+        /// `None` if callable is not bound to any object.
+        pub(super) object_id: Option<u64>,
```

Comment:
Using "linked" here makes the relation obvious.

Also, let's use type-safe `InstanceId` -- or do we really need the state `Some(0)`?
```suggestion
        /// `None` if callable is not linked to any object.
        pub(super) linked_object_id: Option<InstanceId>,
```

Reply:
I also moved "extracting" to dedicated getter

```rs
        /// `None` if callable is not linked with any object.
        pub(super) linked_obj_id: Option<InstanceId>,
    }

    impl<F> FnWrapper<F> {
        pub(crate) fn linked_object_id(&self) -> GDObjectInstanceID {
            self.linked_obj_id.map(InstanceId::to_u64).unwrap_or(0)
        }
    }
```